### PR TITLE
fix: Improve workspace commands

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### What does this PR do?
* Introduced `workspace:stop` command
* `workspace:start` starts a workspace by id
* Added `--debug` flag to debug workspace start
* Refactoring

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/15565
https://github.com/eclipse/che/issues/14873
https://github.com/eclipse/che/issues/16119

### CODE REVIEW

File: src/api/che.ts [hunk 1]
```diff
@@ -199,12 +199,15 @@ export class CheHelper {
     }
   }
 
-  async startWorkspace(cheNamespace: string, workspaceId: string, accessToken?: string): Promise<void> {
+  async startWorkspace(cheNamespace: string, workspaceId: string, debug: boolean, accessToken?: string): Promise<void> {
```
Comment (mmorhun) [inline src/api/che.ts:202]: Maybe we can make `debug` flag defaults to `false` ?

File: src/commands/workspace/create.ts [hunk 2]
```diff
@@ -25,29 +27,50 @@ export default class Create extends Command {
...
+      tasks.add(workspaceTasks.getWorkspaceStartTask(flags.debug))
+    }
+    tasks.add(workspaceTasks.getWorkspaceIdeUrlTask())
 
-    const tasks = this.getWorkspaceCreateTasks(flags)
     try {
-      let ctx = await tasks.run()
-      this.log('\nWorkspace IDE URL:')
+      await tasks.run(ctx)
+      this.log(`Workspace successfully ${flags.start ? 'started' : 'created'}. Workspace IDE URL:`)
```
Comment (mmorhun) [inline src/commands/workspace/create.ts:73]: It's not percise to say `started` as we have only sent a request to start the workspace. it might take a minute until workspace state become `running`.

File: src/commands/workspace/stop.ts [hunk 3]
```diff
@@ -9,36 +9,62 @@
...
-    const tasks = new Listr([
-      { title: 'Verify if we can access Kubernetes API', skip: () => 'Not implemented yet', task: () => { } },
-      { title: 'Verify if Eclipse Che is responding', skip: () => 'Not implemented yet', task: () => { } },
-      { title: 'Verify if the workspaces is running', skip: () => 'Not implemented yet', task: () => { } },
-      { title: 'Stop the workspace', skip: () => 'Not implemented yet', task: () => { } },
-      { title: 'Waiting for the workspace resources to be deleted', skip: () => 'Not implemented yet', task: () => { } },
-    ], { renderer: flags['listr-renderer'] as any })
-
-    await tasks.run()
+    const { args } = this.parse(Stop)
```
Comment (mmorhun) [inline src/commands/workspace/stop.ts:40]: What is the need to mix up flags and args?

File: devfile.yaml [hunk 4]
```diff
@@ -1,110 +0,0 @@
-apiVersion: 1.0.0
```
Comment (mmorhun) [inline devfile.yaml:1]: Why did you remove the devfile ?

Comment (tolusha) [reply devfile.yaml:1]: by mistake

File: src/commands/workspace/create.ts (hunk 2 above)
Comment (tolusha) [reply src/commands/workspace/create.ts:73]: What about `Workspace successfully run` ?

File: src/api/che.ts (hunk 1 above)
Comment (tolusha) [reply src/api/che.ts:202]: Not much advantages

File: src/commands/workspace/stop.ts (hunk 3 above)
Comment (tolusha) [reply src/commands/workspace/stop.ts:40]: For better usability, if some flag is required it is better to use argument instead.

Comment (tolusha) [reply src/commands/workspace/stop.ts:40]: Like other cli tool do

File: src/commands/workspace/create.ts (hunk 2 above)
Comment (mmorhun) [reply src/commands/workspace/create.ts:73]: Please consider: `Workspace has been successfully created. Workspace start request has been sent, workspace will be available shortly`. But up to you.

**Comment by tolusha** [comment]
[test]
